lib/msal-browser/src/config/Configuration.ts

```diff
@@ -38,6 +38,7 @@ export type BrowserAuthOptions = {
     navigateToLoginRequestUrl?: boolean;
     clientCapabilities?: Array<string>;
     protocolMode?: ProtocolMode;
+    azureCloudInstance?: AzureCloudInstance;
```

Comment:
I'm starting to think we should consider breaking authority related options into a separate config object for v3, there's quite a few now

Reply:
Yes, I discussed this before and I strongly think we need to make an `authority` specific config in v3. We can do a doc and add an ADO item in the backlog.